src/LibSCIP.jl

```diff
@@ -1,3 +1,8 @@
+# Copyright (c) 2018 Felipe Serrano, Miles Lubin, Robert Schwarz, and contributors
```

Comment:
we shouldn't add a header here since this is a generated file

Reply:
Removed (also from `gen/prologue.jl`)